pkg/leader/leader.go

```diff
@@ -109,7 +109,33 @@ func Become(ctx context.Context, lockName string) error {
 			log.Info("Became the leader.")
 			return nil
 		case apierrors.IsAlreadyExists(err):
-			log.Info("Not the leader. Waiting.")
+			existing := &corev1.ConfigMap{}
+			key := crclient.ObjectKey{Namespace: ns, Name: lockName}
+			err := client.Get(ctx, key, existing)
+			if err != nil {
+				return err
+			}
+			leaderPod := &corev1.Pod{}
+			key = crclient.ObjectKey{Namespace: ns, Name: existing.ObjectMeta.OwnerReferences[0].Name}
+			err = client.Get(ctx, key, leaderPod)
+			if err != nil {
+				return err
+			}
+
+			podFailed := leaderPod.Status.Phase == corev1.PodFailed
+			podEvicted := leaderPod.Status.Reason == "Evicted"
+			if podFailed && podEvicted {
+				log.Info("Operator pod with leader lock has been evicted.", "Leader pod", leaderPod.Name)
```

Comment:
```suggestion
				log.Info("Operator pod name %s is with leader lock and has been evicted.", leaderPod.Name)
```

Reply:
@camilamacedo86  I believe your suggestion would result in a panic. This function does not support printf-style templating and must have an odd number of arguments (one for the static message, and then any number of key/value pairs). 

Please see the relevant discussion [here](https://github.com/operator-framework/operator-sdk/issues/2186#issuecomment-553425503) and the [documentation for the `logr.Info()` function](https://github.com/go-logr/logr/blob/a1ebd699b1950beb1da0752cbb5559662018f798/logr.go#L124-L130).